### ISSUE TITLE
NPN Error - No matching version found for google-tts-api@0.0.3

### ISSUE THREAD

**Issue by Paul-Reed** [Hypnos3/node-red-contrib-cast#3]
Looking in the NPM log, I'm getting this error when attempting to update to v0.0.3, and the update fails.
I'm assuming that it's because the [current google-tts-api npm version is v0.0.2](https://www.npmjs.com/package/google-tts-api), but you are requesting v0.0.3 which isn't available.
```
30 verbose argv "/usr/bin/node" "/usr/bin/npm" "install" "--save" "--save-prefix=~" "--production" "node-red-contrib-cast@0.0.3"
31 verbose node v8.11.4
32 verbose npm  v6.4.0
33 error code ETARGET
34 error notarget No matching version found for google-tts-api@0.0.3
35 error notarget In most cases you or one of your dependencies are requesting
35 error notarget a package version that doesn't exist.
35 error notarget
35 error notarget It was specified as a dependency of 'node-red-contrib-cast'
36 verbose exit [ 1, true ]
```

**Comment by jbkuma** [comment]
same error for me

**Comment by Hypnos3** [comment]
fixed with version 0.0.4 (still not working, caused by open issues)

**Comment by Hypnos3** [comment]
0.0.6 should now work.
